openmls/tests/test_managed_group.rs

```diff
@@ -258,43 +243,76 @@ fn managed_group_operations() {
             let queued_message = alice_group
                 .create_message(&crypto, message_alice)
                 .expect("Error creating application message");
-            let events = bob_group
-                .process_message(queued_message.into(), &crypto)
-                .expect("The group is no longer active");
-
-            // Check that we received the correct event
-            match events.last().expect("Expected an event to be returned") {
-                GroupEvent::ApplicationMessage(application_message_event) => {
-                    assert_eq!(application_message_event.sender(), &alice_credential);
-                    assert_eq!(application_message_event.message(), message_alice);
-                }
-                _ => unreachable!("Expected an ApplicationMessage event"),
+
+            let unverified_message = bob_group
```

Comment:
Would it be worth to introduce a combined parse-process function? Even if just for testing purposes?

Reply:
The thought had crossed my mind. It really only affects the tests, because I don't think we should expose that to consumers (they should be forced to inspect processed messages). I don't have a strong opinion about it though, if you think it's worth it, I'll put it in.

Reply:
No worries. It's effectively already in the test framework, so we should be fine. And should it lead to extensive code duplication in the future, we can still add it then.